engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/MainMenuScreen.java

```diff
@@ -56,7 +56,7 @@ public void initialise() {
         WidgetUtil.trySubscribe(this, "settings", button -> triggerForwardAnimation(SettingsMenuScreen.ASSET_URI));
         WidgetUtil.trySubscribe(this, "credits", button -> triggerForwardAnimation(CreditsScreen.ASSET_URI));
         WidgetUtil.trySubscribe(this, "exit", button -> engine.shutdown());
-        WidgetUtil.trySubscribe(this, "crashReporter", widget -> CrashReporter.report(new Throwable("Report an error."), LoggingContext.getLoggingPath()));
+        WidgetUtil.trySubscribe(this, "crashReporter", widget -> CrashReporter.report(new Throwable("Report a error."), LoggingContext.getLoggingPath(), false));
```

Comment:
I believe "Report an error" is more correct in this case than "Report a error" - English is weird that way. You can *generally* go with `an` if the next word starts with a vovel and otherwise `a` but there are exceptions ...